### ISSUE TITLE
openmediavault flashmemory plugin can't activated

### ISSUE THREAD

**Issue by Nibeck1309** [OpenMediaVault-Plugin-Developers/openmediavault-flashmemory#4]
hello when I want to activated the flashmomory plugin I have this error:

Erreur #4000:
exception 'OMVException' with message 'Failed to execute command 'omv-mkconf flashmemory 2>&1': 
/usr/share/openmediavault/mkconf/flashmemory: 66: /usr/share/openmediavault/mkconf/flashmemory: #!/bin/bash
# waiting for the package manager to finish its job.

```
while [[ ! -f /var/lib/dpkg/lock ]]; do
sleep 1
done
```
# disabling apt disk cache so its folder in /var/cache/apt/whatever will stop filling up. Also saves additional writes.

```
echo Dir::Cache ""; >> /etc/apt/apt.conf.d/02nocache
echo Dir::Cache::archives ""; >> /etc/apt/apt.conf.d/02nocache
```
# disabling cron-apt package download by erasing the file asking for it.

```
rm /etc/cron-apt/action.d/3-download
```
# deleting the packages already in the cache, because otherwise this is all wasted space in the tmpfs
# and time wasted on startup and shutdown as all this stuff is compressed/extracted, especially for devices with weak processors

```
rm /var/cache/apt/archives/*
rm /var/cache/apt/archives/partial/*
```
# script self-destruct, deletes this script from disk. technically optional, as /tmp content gets dumped anyway on startup.

```
rm /tmp/fs2ramhousekeeper
```

exit 0 : not found' in /usr/share/openmediavault/engined/module/flashmemory.inc:59
Stack trace:
#0 /usr/share/openmediavault/engined/rpc/config.inc(180): OMVModuleFlashmemory->applyConfig()
#1 [internal function]: OMVRpcServiceConfig->applyChanges(Array, Array)
#2 /usr/share/php/openmediavault/rpcservice.inc(125): call_user_func_array(Array, Array)
#3 /usr/share/php/openmediavault/rpcservice.inc(158): OMVRpcServiceAbstract->callMethod('applyChanges', Array, Array)
#4 /usr/share/openmediavault/engined/rpc/config.inc(224): OMVRpcServiceAbstract->callMethodBg('applyChanges', Array, Array)
#5 [internal function]: OMVRpcServiceConfig->applyChangesBg(Array, Array)
#6 /usr/share/php/openmediavault/rpcservice.inc(125): call_user_func_array(Array, Array)
#7 /usr/share/php/openmediavault/rpc.inc(79): OMVRpcServiceAbstract->callMethod('applyChangesBg', Array, Array)
#8 /usr/sbin/omv-engined(500): OMVRpc::exec('Config', 'applyChangesBg', Array, Array, 1)
#9 {main}

Thanks

**Comment by ryecoaaron** [comment]
Fixed in 1.7 in the repo now.

**Comment by Nibeck1309** [comment]
thanks
